Sources/Bootstrap/Tags/ButtonGroup.swift

```diff
@@ -0,0 +1,70 @@
+import Leaf
+import TemplateKit
+
+/// Bootstrap Button Group Tag
+public final class ButtonGroupTag: TagRenderer {
+    enum GroupKeys: String {
+        case standard = "btn-group"
+        case vertical = "btn-group-vertical"
+    }
+
+    private static let paramCount: Int = 3
+
+    public func render(tag: TagContext) throws -> Future<TemplateData> {
+
+        let body = try tag.requireBody()
+
+        try tag.requireParameterCount(upTo: ButtonGroupTag.paramCount)
+
+        var group = GroupKeys.standard.rawValue
+        var classes: String?
+        var aria = UUID().uuidString
+
+        if tag.parameters.count > 0 {
+            guard let param = tag.parameters[0].bool else {
+                throw tag.error(reason: "Wrong type given (expected a bool): \(type(of: tag.parameters[0]))")
+            }
+
+            if param {
+                group = GroupKeys.vertical.rawValue
+            }
+        }
+
+        if tag.parameters.count > 1 {
+            guard let param = tag.parameters[1].string else {
+                throw tag.error(reason: "Wrong type given (expected a string): \(type(of: tag.parameters[0]))")
```

Comment:
It should be `tag.parameters[1]` in the thrown error.

Reply:
Will fix those. I think that is the same in alert tag